kubectl-rook-ceph.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+usages() {
+  cat <<EOF
+Description:
+  kubectl rook-ceph provides common management and troubleshooting tools for Rook-Ceph.
+
+Commands:
+  ceph - run any 'ceph' CLI command as given
+    args  - any 'ceph' CLI arg (e.g., status)
+    flags - any 'ceph' CLI flag (e.g., --format=json-pretty)
+
+Usage:
+  kubectl rook-ceph [command] [command args...] [command flags...]
```

Comment:
What's the difference between args and flags?

Reply:
example:
`ceph status -f json`

so `status` is `ceph` command `arg` here `-f json` is flag. Right @BlaineEXE ?

Reply:
I would say that's correct, yes. I also don't think this is worth a lot of nit-picking.

In some sense, they are all `args`, and those can be `(sub)commands` or `flags`.

But from the standpoint of the tool `ceph` is a `command`, and then everything that comes after is irrelevant to us, so we could just say `args`.

So... maybe it's simpler for users to just say `[command args...]`. Then we don't have to be as specific. 🤷‍♀️

Reply:
The example does not match https://github.com/rook/kubectl-rook-ceph/pull/4/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR54

`kubectl rook ceph foo`

Reply:
changed.